### PR TITLE
Add feedback in Room meeting is running

### PR DESCRIPTION
**Pull request by scouillard**
## Description

Add feedback in Room if the meeting is running.
Create an "online pill" as feedback.
Thinking about adding this pill to the Room card in Rooms as well.
image

## Testing Steps

Start/End a meeting and see the change in Room.

### CODE REVIEW

File: app/javascript/components/rooms/room/Room.jsx [hunk 1]
```diff
@@ -19,7 +22,9 @@ export default function Room() {
   const { friendlyId } = useParams();
   const { isLoading, data: room } = useRoom(friendlyId);
   const startMeeting = useStartMeeting(friendlyId);
+  const { isLoadingRunning, data: isRunning } = useMeetingRunning(friendlyId);
```
Comment (KH-Amir-TN) [inline app/javascript/components/rooms/room/Room.jsx:25]: I suspect that `isLoadingRunning` here is `undefined`?
To rename it I suggest the use of the following syntax `const {isLoading: isLoadingRunning, data: isRunning} = useMeetingRunning(...);`

Comment (KH-Amir-TN) [reply app/javascript/components/rooms/room/Room.jsx:25]: Or maybe avoid destructing the returned value and use namespaces `const meetingRunningQuery = useMeetingRunning(...);`

File: app/javascript/components/rooms/room/Room.jsx [hunk 2]
```diff
@@ -31,13 +36,25 @@ export default function Room() {
...
       </Row>
       <Row className="my-5">
-        <Col>
-          <h3>{room.name}</h3>
+        <Col className="col-xxl-8">
+          <Stack direction="horizontal" gap={2}>
+            <h1>{room.name}</h1>
+            { isRunning
+              && (
+                <Badge className="rounded-pill online-pill ms-2 text-success">
```
Comment (KH-Amir-TN) [inline app/javascript/components/rooms/room/Room.jsx:44]: Can we extract this into a shared component?
That will help us re-using it,

Comment (scouillard) [reply app/javascript/components/rooms/room/Room.jsx:44]: It is changed into a component in the following PR #3938

**Review by farhatahmad — APPROVED** [review]

**Pull request merged**